src/ompl_planner.cpp

```diff
@@ -187,9 +195,9 @@ OMPLPlannerTpl<DATATYPE>::plan(VectorX const &start_state, std::vector<VectorX>
 
     planner->setProblemDefinition(pdef);
     planner->setup();
-    if (verbose)
-        std::cout << "OMPL setup" << std::endl;
+    std::cerr << "OMPL setup" << std::endl;
     ob::PlannerStatus solved = planner->ob::Planner::solve(time);
+    std::cerr << "OMPL solved" << std::endl;
```

Comment:
segfaults before this line...

Reply:
In particular, this is the stdout
> OMPL setup
[1]    955667 segmentation fault (core dumped)  python3.11 examples/collision_avoidance.py